ADMVEDU95/ADMVEDU95/BaseAuthPresenter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BasePresenter.swift
+//  ADMVEDU95
+//
+//  Created by Satishur, Oleg on 20.05.2021.
+//
+
+import Foundation
+import FirebaseAuth
+
+class BaseAuthPresenter {
+    var firebaseService: FirebaseServiceProtocol?
```

Comment:
let firebaseService and set via init
and in subclasses super.init should be called